packages/@aws-cdk/core/lib/cfn-parse.ts

```diff
@@ -110,11 +122,11 @@ export class FromCloudFormation {
     return ret;
   }
 
-  public static parseCreationPolicy(policy: any): CfnCreationPolicy | undefined {
+  public static parseCreationPolicy(policy: any, options: ParseCfnOptions): CfnCreationPolicy | undefined {
```

Comment:
Passing around all the options everywhere feels a little dirty to me.

I'd think I'd prefer a class `CfnParser` or something which has the resolver as a member.

Not going to force you to change it, but I have a feeling it will be cleaner. Give it a thought.

Reply:
Done. @rix0rrr let me know if you like this way better.